src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -11027,6 +11065,45 @@ definitions:
          format: date-time
       values:
         type: object
+  ActionConstraint:
+    description: |
+      Response object holding action parameter constraint name and map of its values-labels.
```

Comment:
I assume the map is from labels to values?

Reply:
See my other comment regarding that map.